website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -24,26 +24,69 @@ HashiCorp helm repository and ensure you have access to the chart:
 
 ```shell-session
 $ helm repo add hashicorp https://helm.releases.hashicorp.com
+
 "hashicorp" has been added to your repositories
+```
 
+```shell-session
 $ helm search repo hashicorp/vault-secrets-operator
+
 NAME           	CHART VERSION	APP VERSION	DESCRIPTION
 hashicorp/vault-secrets-operator	0.3.1       	0.3.1     	Official HashiCorp Vault Secrets Operator Chart
 ```
 
 Then install the Operator:
 
 ```shell-session
-$ helm install --create-namespace --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator --version 0.3.1
+$ helm install --version 0.3.1 --create-namespace --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator
 ```
 
 
 ## Upgrading using helm
 
-Upgrades may be performed with `helm upgrade` on an existing install. Please
+Upgrades may be performed with `helm upgrade` on an existing installation. Please
 always run Helm with `--dry-run` before any install or upgrade to verify
 changes.
 
+Update the `hashicorp` Helm repo.
+```shell-session
+$ helm repo update hashicorp
+
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "hashicorp" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+Upgrade the VSO release
+```shell-session
+$ helm upgrade --version 0.3.1 --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator
+```
+
+~> **Note:** Helm does not automatically update the Operator's CRDs.
+All CRDs will need to be updated manually after upgrading VSO.
+Refer to [Updating the CRDs](#updating-the-crds) for the proper update procedure.
```

Comment:
```suggestion
<Note title="Helm does not automatically update CRDs">

You must update all CRDs manually after upgrading VSO. Refer to
the  [Updating CRDs](#updating-the-crds) section for details.

</Note>
```
Style correction: write in active voice, address the reader when providing instruction, use new aside notation

Reply:
Fixed in https://github.com/hashicorp/vault/pull/23512/commits/f088661672eb9de9cea37ec46f670c893fdebd7d

Please note that Markdown syntax does not work in an HTML tag. I updated your suggestion.